src/stories/Library/Modals/modal-loan/modal-loan.scss

```diff
@@ -47,4 +47,14 @@ $MODAL_LOAN_CONTAINER_WIDTH: 1000px;
   justify-content: flex-end;
   align-items: center;
   margin-top: 40px;
+
+  &--bottom {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    background-color: $c-global-primary;
+    padding: 20px 0;
```

Comment:
This is just a note but I think the optimal solution to spacing properties would be to have some fixed variables/mixins for them

Reply:
Oh yeah, usually I remember, Ill fix it riddaway